tests/agent-scenarios.test.ts

```
@@ -0,0 +1,351 @@
+  }
+
+  if (expected.volume && actionCall.parameters.volume !== expected.volume) {
+    validations.push(`Expected volume ${expected.volume}, got ${actionCall.parameters.volume}`);
+  }
+
+  if (expected.activity_id && actionCall.parameters.activityId !== expected.activity_id) {
+    validations.push(`Expected activity_id '${expected.activity_id}', got '${actionCall.parameters.activityId}'`);
+  }
+
+  // Check if time parsing happened when expected
+  if (expected.start_time && expected.start_time.includes('ago')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)